core/button_view.js

```diff
@@ -21,24 +21,15 @@ function ButtonView(options) {
 util.inherits(ButtonView, TextView);
 
 ButtonView.prototype.onKeyPress = function(ch, key) {
-    if(this.isKeyMapped('accept', key.name) || ' ' === ch) {
+    let actionForKeyName = key ? key.name : ch;
```

Comment:
super nit: `const`

Reply:
Fixed - since it was only used once went one further and just got rid of the unnecessary variable altogether.